### DISCUSSION TITLE
Enabling GitHub Discussions on your repository

### DISCUSSION THREAD

**Discussion by imanoar** [tarikmanoar/tarikmanoar#5] [Q&A] [answered]
Repository owners and people with write access can enable GitHub Discussions for a community on their public and private repositories.

When you first enable GitHub Discussions, you will be invited to configure a welcome post.

**Comment by tarikmanoar** [comment, marked as answer]
On GitHub.com, navigate to the main page of the repository.
Under your repository name, click  Settings.
Under "Features", click Set up discussions.
Under "Start a new discussion," edit the template to align with the resources and tone you want to set for your community.
Click Start discussion.
